src/golang/cmd/migrator/versions/000016_add_artifact_type_column_to_artifact/utils.go

```diff
@@ -257,7 +257,8 @@ func migrateArtifact(ctx context.Context, db database.Database) error {
 			// and we need to revert the content change.
 			originalContent, err := storage.NewStorage(storageConfig).Get(ctx, artifactResult.ContentPath)
 			if err != nil {
-				return err
+				log.Infof("Skipping data migration for artifact result %s since its content wasn't generated", artifactResult.Id)
```

Comment:
I think we should check if `err == [specific error] && status != 'succeeded'` because if someone encounters a different error at this step, we wouldn't know.

Reply:
will update, thx!